README.md

```diff
@@ -1,47 +1,95 @@
 # shadowcat
 
-Invisible web component auth implementation of Manifold&#39;s PUMA &lt;https://github.com/manifoldco/puma&gt;
+Simple JS OAuth solutions for [Manifold UI][manifold-ui]
 
 ## Usage
 
-1. `npm i @manifoldco/shadowcat`
-2. Import and register the `manifold-oauth` web component:
+### React
 
-```js
-import { defineCustomElements } from '@manifoldco/shadowcat/dist/loader';
+In a terminal, install `@manifoldco/shadowcat` for your app:
 
-defineCustomElements(window);
+```bash
+npm i @manifoldco/shadowcat
 ```
 
-3. Add the following code anywhere on your website/app to get an auth token:
+Then register the web component (we recommend as high up the DOM tree as possible, so it’ll load sooner).
+
+```jsx
+import React, { useEffect, useRef } from "react";
+
+import("@manifoldco/shadowcat/dist/loader").then(({ defineCustomElements }) =>
+  defineCustomElements(window)
+);
+
+const App = () => {
+  const authRef = useRef(null);
+
+  useEffect(() => {
+    // Fires whenever a token is received
+    const tokenReceived = detail => {
+      console.log(detail);
+      // {
+      //   duration: 1450,
+      //   error: null,
+      //   expiry: 1566565841,
+      //   token: "Hi I'm a super duper secret token!"
+      // }
+    };
+
+    if (authRef.current) {
+      authRef.addEventListener("receiveManifoldToken", tokenReceived);
+    }
+  }, []);
+
+  return (
+    <>
+      <Routes />
+      <manifold-oauth ref={authRef} /> {/* this can be placed anywhere, really, but the sooner it loads the better  */}
+    </>
+  );
+};
+```
+
+### HTML / JS
 
 ```html
-<manifold-oauth oauth-url="[your-oauth-url]"></manifold-oauth>
+<body>
+  <manifold-auth></manifold-auth>
+  <script src="https://unpkg.com/@manifoldco/shadowcat/dist/manifold.js"></script>
+  <script>
+    document
+      .querySelector("manifold-auth")
```

Comment:
I think this is a typo, should be `.querySelector("manifold-oauth")` right?

Reply:
Oh you’re right. Will fix.